factory/defaults/chains/testnet.yaml

```diff
@@ -298,3 +298,16 @@ chains:
     chain_name: xrp (Devnet)
     explorer_url: https://testnet.xrpl.org/
     decimals: 6
+  BASE:
+   chain: BASE
+   driver: evm
+   chain_name: Base Protocol
+   decimals: 18
+   explorer_url: https://explorer.baseprotocol.com
+  NOBL:
+   chain: NOBL
+   driver: cosmos
+   chain_name: Noble
+   chain_prefix: noble
```

Comment:
cosmos chain assets always have a contract address so we need to configure one.  noble doesn't seem to have a "noble" asset similar to other cosmos chains, (not that I can find here https://explorer.nodestake.org/noble/supply).  See other cosmos chains.

Instead we can use one of the tokens, like `uusdc`.  This means crosschain will use uusdc for pay for gas.  We may need to update later to support choosing between available assets to pay for gas later on.

```
    chain_coin: uusdc
```